app/src/main/java/com/infomaniak/drive/utils/Extensions.kt

```diff
@@ -278,24 +286,29 @@ fun View.setFileItem(file: File, isGrid: Boolean = false) {
 
     filePreview.scaleType = ImageView.ScaleType.CENTER
 
+    fun tintedDrawable(@IdRes iconId: Int, color: String? = null): Drawable? {
+        return ContextCompat.getDrawable(context, iconId)?.apply {
+            setTintList(ColorStateList.valueOf((color ?: file.getColor(context)).toColorInt()))
+        }
+    }
+
```

Comment:
It should be in Utils.kt.

Reply:
For now, this function is only used here.

Reply:
Done in another PR: https://github.com/Infomaniak/android-kDrive/pull/481